packages/mobile/e2e/src/usecases/ExchangeCelo.js

```diff
@@ -91,10 +94,15 @@ export default ExchangeCelo = () => {
     await enterPinUiIfNecessary()
     // Get transaction time
     let transactionTime = new Date()
-    // Scroll to top of feed
-    await element(by.id('ExchangeScrollView')).scroll(200, 'down')
     // Wait 10 seconds checking that error banner is not visible each second
     await waitForExpectNotVisible('errorBanner')
+    // Scroll to top of feed - scroll not possible if not all transactions loaded
+    try {
+      await waitFor(element(by.id('ExchangeScrollView')))
```

Comment:
i just noticed that this same code block is used a few times, could we abstract to a helper function?

Reply:
Should be fixed in https://github.com/valora-inc/wallet/pull/2347/commits/ec44ddb31add2ccb3394211fad65e43d8cd27fd9!